index.html

```diff
@@ -5,11 +5,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+    <link rel="stylesheet" href="style.css">
     <title>Document</title>
 </head>
 
+
 <body>
-    <h1>My heading</h1>
+    <header>
+        <h1>COLORADO LIFE</h1>
```

Comment:
Remember to achieve all-caps text via `text-transform` in CSS and not in the markup